dist/css/style.css

```diff
@@ -82,6 +82,14 @@ li {
   color: #172b4d;
 }
 
+@media (min-width: 768px) {
+  .title--3 {
+    font-weight: 800;
+    font-size: 48px;
+    line-height: 60px;
```

Comment:
Great Job, for sure you are going to be a great developer.
In order to make more responsive your designs you could think in use rem instead of pixels since pixels are absolute values and rem works better in responsive designs because they are relative.

Reply:
Thanks. @dgcuenca.